doc/classes/GraphEdit.xml

```diff
@@ -172,7 +174,7 @@
 			<param index="1" name="max_distance" type="float" default="4.0" />
 			<description>
 				Returns the closest connection to the given point in screen space. If no connection is found within [param max_distance] pixels, an empty [Dictionary] is returned.
-				A connection consists in a structure of the form [code]{ from_port: 0, from_node: "GraphNode name 0", to_port: 1, to_node: "GraphNode name 1" }[/code].
+				A connection consists in a structure of the form [code]{ from_port: 0, from_node: "GraphNode name 0", to_port: 1, to_node: "GraphNode name 1", keep_alive: true/false }[/code].
```

Comment:
Why did you write specific example values for ports and nodes, but `true/false` for alive?